apps/extension/src/background/keyring/messages.ts

```diff
@@ -1,11 +1,16 @@
-import { Message } from "../../router";
+import { PhraseSize } from "@anoma/crypto";
+import { Message } from "router";
 import { ROUTE } from "./constants";
-import { KeyRingStatus } from "./keyring";
+import { Bip44Path, DerivedAccount, KeyRingStatus } from "./types";
 
 enum MessageTypes {
```

Comment:
nit: Not part of this PR but maybe better to keep enums in singular naming

```suggestion
enum MessageType {
```